rpc/src/main/java/com/distkv/rpc/protobuf/distkv_pb.proto

```diff
@@ -67,6 +69,7 @@ message DistkvRequest {
     string key = 1;
     RequestType requestType = 2;
     google.protobuf.Any request = 3;
+    string name = 4;
```

Comment:
Why should we change this line? Is this used for ParsedResult ?

Reply:
Yes. I used for the parsed result